Lineup.cpp

```diff
@@ -0,0 +1,45 @@
+#include "Lineup.h"
+
+
+Lineup::Lineup()
+{
+    std::cout << "a new Lineup was created" << std::endl;
+}
+
+Lineup::~Lineup()
+{
+    std::cout << "a Lineup was destroyed" << std::endl;
+    tuesday.isBusy = false;
+}
+
+void Lineup::getSurferInfo( SurfReport::Surfer surfer ) 
+{
+    if ( surfer.atBeach == true)
+    {
+        std::cout << "surfer is at the beach" << std::endl;
+    }
+    else 
+    {
+        std::cout << "surfer is not at the beach" << std::endl;
+    }
+    std::cout << "surfer is " << surfer.surferAge << " years old" << std::endl;
+
+    std::cout << "surfer's skill level is: " << surfer.surferSkill << std::endl;
+    
+    std::cout << "surfer is using a " << surfer.boardSize << "ft board" << std::endl;
+    
+    if ( surfer.havingLesson == true)
+    {
+        std::cout << "surfer is having lesson" << std::endl;
+    }
```

Comment:
the `== true` is not needed because the member variable being compared is a `bool`.
- [ ] revise